Gemfile

```diff
@@ -89,6 +89,8 @@ gem 'redis', '>= 4.0', require: ['redis', 'redis/connection/hiredis']
 gem 'sidekiq'
 gem 'sidekiq-scheduler'
 
+gem "cable_ready", "5.0.0.pre2"
```

Comment:
Finally? 🎉

Reply:
CableReady and StimulusJS are both small components of the StimulusReflex stack. But we still haven't added the whole thing, we're not using Websockets or Reflexes yet. Small steps!